test/jdk/javax/swing/JScrollPane/AcceleratedWheelScrolling/RTLScrollers.java

```diff
@@ -0,0 +1,694 @@
+/*
+ * Copyright (c) 2004, 2025, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+// A few Swing components which use the mouse wheel to scroll
+//
+// NOTE: The robot portion of this test probably has a number of EDT problems,
```

Comment:
Yikes. You know that there are EDT problems but just decided to ignore them. Are there a lot of them? How hard it would be to get rid of them?

Reply:
That's a remnant from the original test. I tried cleaning up the robot calls and test methods further. Re-tested `HorizScrollers` on my windows machine and all still looks good. I don't see how it's feasible to call all the methods in EDT in this JDialog helper class. So far the test looks ok and I have not encountered any issues, but let me know if you have any suggestions or concerns.

Reply:
I made what UI elements I could static and volatile to add methods that change UI elements to the EDT. Test still runs as expected and passes on W11.